app/Http/Controllers/PhpReboot/HomeController.php

```diff
@@ -14,6 +17,29 @@ class HomeController extends Controller
      */
     public function homePage()
     {
-        return view('PhpReboot.Home.homePage', ['menu' => 'home']);
+        $cacheTimeout = 1440;
+        
+        // Number of Magazines | Stores it in the cache
+        $issuesCount = Cache::remember('issueCount', $cacheTimeout, function(){
+           return Magazine::count(); 
+        });
+        
+        //Number of Articles
+        $articlesCount = Cache::remember('articlesCount', $cacheTimeout, function(){
+            return Article::count();
+        });
+        
+        //Last Article Date
+        $lastArticleDate = Cache::remember('lastArticleDate', $cacheTimeout, function(){
+            $lastArticle = Article::orderBy('published_at', 'Desc')->first();
+            return $lastArticle->published_at;
+        });
```

Comment:
Hi @kapilsharma ,
No problem with that. I am glad you pointed out the mistakes. Thank you very much for the review. I will surely take them into consideration next time. I will be waiting to see your solution.
Now, i will make a new PR to issue-115 branch.

Happy Diwali !!!